tests/end_to_end/candid_rpc/class_syntax/stable_structures/src/canister1/index.ts

```diff
@@ -1,42 +1,251 @@
-import { IDL, query } from 'azle';
+import { convertOpt, IDL, query, StableBTreeMap, update } from 'azle';
 
-import { StableMap0Methods } from './stable_map_0';
-// import { stableMap1Methods } from './stable_map_1';
-// import { stableMap2Methods } from './stable_map_2';
-// import { stableMap3Methods } from './stable_map_3';
-// import { stableMap4Methods } from './stable_map_4';
+import { Reaction, User } from '../types';
 
-// let redeployed = false;
+let stableMap0 = StableBTreeMap<number, string>(0);
+let stableMap1 = StableBTreeMap<number, Uint8Array>(1);
+let stableMap2 = StableBTreeMap<number, bigint>(2);
+let stableMap3 = StableBTreeMap<Reaction, bigint>(3);
+let stableMap4 = StableBTreeMap<User, number>(4);
 
-export default class extends StableMap0Methods {
+export default class {
     @query([], IDL.Bool)
     getRedeployed(): boolean {
-        return globalThis._azlePostUpgradeCalled;
+        return globalThis._azlePostUpgradeCalled; // TODO this should be set in the Rust init and post upgrade so that they are always set
```

Comment:
get rid of this TODO

Reply:
This has been removed in a subsequent PR